Dependencies/babylon_native_utils/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+set(SOURCES
+    "Include/Babylon/NetworkUtils.h"
+    "Include/Babylon/ticketed_collection.h"
+    "Source/NetworkUtils.cpp")
+
+add_library(BabylonNativeUtils ${SOURCES})
+
+target_include_directories(BabylonNativeUtils PRIVATE "Include/Babylon")
+target_include_directories(BabylonNativeUtils INTERFACE "Include")
+
+target_link_libraries(BabylonNativeUtils
+    PUBLIC arcana
```

Comment:
Is it expected that arcana is already added?

Reply:
Yes. The pattern being established here is that plugins (and modules following the plugin pattern) do not supply their own dependencies, but instead assume they exist as a part of a larger build system that bears the responsibility of making sure all dependencies are loaded in the right order. This allows the inclusions to scale horizontally and, when applicable, share dependencies amongst themselves.